Sources/GrimoireKit/File.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+struct File {
+  static func read(_ path: String) throws -> String {
+    let path = "file://" + path
+    guard let url = URL(string: path) else {
+      throw Error.invalidFilePath(path)
+    }
+    return try String(contentsOf: url, encoding: .utf8)
+  }
+}
+
+extension File {
+  enum Error: LocalizedError {
+    case invalidFilePath(String)
+
+    var errorDescription: String? {
+      switch self {
+      case .invalidFilePath(let path):
```

Comment:
I'm team `case let .invalidFilePath(path):` but I don't mind if you prefer that way.